assets/javascripts/discourse/templates/modal/community-ui-builder.hbs

```diff
@@ -35,6 +35,16 @@
         }}
       </section>
 
+      <section class="field">
+        <label>{{i18n "communitarian.community.ui_builder.community_code.label"}}</label>
+        {{text-field
+              value=model.custom_fields.community_code
+              autocomplete="discourse"
+              maxlength="5"
```

Comment:
I think that 5 characters will not be enough for some users. Maybe it will be better to change it to 10 OR move to settings?